### PR TITLE
Improve dashboard UI to prepare for grouping repos by org.

### PR DESCRIPTION
**Pull request by frechg**
screen shot 2015-02-27 at 12 55 27 pm

### CODE REVIEW

File: app/assets/stylesheets/application/_header.scss [hunk 1]
```diff
@@ -1,10 +1,23 @@
 $medium-screen-header: new-breakpoint(max-width em(850) 4);
 
-header {
-  @include clearfix;
+header.global {
+  @include outer-container;
```
Comment (frechg) [inline app/assets/stylesheets/application/_header.scss]: @salbertson Hound is wrong here. The SCSS follows the rules Hound states.

**Comment by salbertson** [comment]
@frechg if those nesting comments aren't useful go ahead and update the SCSS config so they are in the future.

**Comment by salbertson** [comment]
Looks great! :+1:

**Comment by salbertson** [comment]
@frechg, wasn't this merged? Should the PR be closed?

**Comment by teoljungberg** [comment]
The repo-refresh button looks like an input field more than a button, is that intentional @frechg ?

**Comment by croaky** [comment]
Is the repo-refresh button necessary? Could we refresh repos automatically?

**Pull request closed without merging**